mavisp/core.py

```diff
@@ -209,7 +216,8 @@ def ingest(self, stop_at='critical'):
             mode = r['mode']
             mutations = r['mutations']
             curators = r['curators']
-
+            uniprot_ac= r["uniprot_ac"]
+            refseq_d= r["refseq_d"]
             if mutations is None:
                 mavisp_criticals.append(MAVISpCriticalError("the mutation list was not available, readable or in the expected format"))
             if curators is None:
```

Comment:
here you should add conditions for `refseq_id` and `uniprot_ac` - if they are `None` it means they were not available and so the entry is not even processed